src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigAlignmentsConfigPicker.java

```diff
@@ -460,12 +595,41 @@ private static AssemblyContigWithFineTunedAlignments updateContigMappingsWithGap
         final Comparator<AssemblyContigWithFineTunedAlignments> mismatchSecond
                 = (AssemblyContigWithFineTunedAlignments x, AssemblyContigWithFineTunedAlignments y)
                 -> Integer.compare(x.getAlignments().stream().mapToInt(ai -> ai.mismatches).sum(),
-                                   y.getAlignments().stream().mapToInt(ai -> ai.mismatches).sum());
+                y.getAlignments().stream().mapToInt(ai -> ai.mismatches).sum());
         return numFirst.thenComparing(mismatchSecond);
     }
 
+    // functionality group 3.1: split gaps
+
+    /**
+     * Split the good alignments stored in {@code configuration}.
+     *
+     * Note that an edge case is possible where for some gapped alignment,
+     * after gap-split, one or more child alignments might be contained by
+     * another original alignment.
+     *
+     * Example:
+     * picture: ref span by gapped original alignment   -------------          --------
+     *          ref span by overlapping other alignment                  ---------------------------
+     *
+     * TODO:
```

Comment:
Can you expand on what the TODO is in the comment? Ie. "choose one of these options"..